quarto/src/Api/Piece.php

```diff
@@ -0,0 +1,15 @@
+<?php
+namespace App\Api;
+
+
+class Piece {
+
+    public $id;
```

Comment:
It's a bad practice to make public properties. Please use private or protected ones with getters / setters (getters only if you're using a [ValueObject](https://en.wikipedia.org/wiki/Value_object))